### ISSUE TITLE
Feature Suggestion: Dashboard Tile

### ISSUE THREAD

**Issue by AmyAmy** [rstedman/tfs-pullrequest-dashboard#4]
Thanks for this awesome plugin.

I have a suggestion for a feature that would make it even more useful:

How about a [dashboard](https://www.visualstudio.com/en-us/docs/report/dashboards) tile, showing the number of open PRs, (like the 1×1 query tiles), which, when clicked, opens the pullrequest dashboard page.

**Comment by rstedman** [comment]
How do you picture this tile working?  Just a count of all PRs across all repos?  In my mind I'd want this filtered down to a count of PRs I care about - for example maybe an option of showing a count of PRs filtered by assigned to me/assigned to my team/created by me/all (dropdown select in tile settings).  

Would this suit what you're looking for?  If not, could you provide more detail on your use case?

**Comment by AmyAmy** [comment]
For me the most important thing is to remind my team that there are PR's to review, so "assinged to my team", or "all" would be the nicest to have. A dropdown selecting the various options you described above would be a nice extra.

Perhaps it would be possible to add a checkbox to only count the PRs that haven't been accepted by the current user yet, so it's immediately clear if there is current action required by the user?

In the end the most important thing would be to show the actual dashboard when the user clicks the tile.

Another nice extra, but certainly not needed would be to have some color-coding, change the color if there are PRs that don't have a response from the current user yet.

I'd imagine it would look something like this:

![image](https://user-images.githubusercontent.com/630387/31116852-d4ca5dae-a827-11e7-9620-3ff6c3b081f1.png)

**Comment by rstedman** [comment]
I've release 2.0.0 of this extension, which introduces a new VSTS widget.  I went a different direction from your suggestion of a tile widget with just a count.  Instead, I chose to model the widget after the "Pull Requests" widget provided by MS, but extending it to show across all repos.  I felt that this would be more useful than just a count.

ex:

![image](https://user-images.githubusercontent.com/5440997/34081661-7f825c66-e31e-11e7-98d6-fda244c4b6aa.png)

I realize this isn't what you suggested, but does it meet what you are looking for?
